internal/test_cases/tokenize_test_case.go

```diff
@@ -74,3 +59,21 @@ func (t *TokenizeTestCase) Run(executable *interpreter_executable.InterpreterExe
 
 	return nil
 }
+
+func logReadableFileContents(logger *logger.Logger, fileContents string) {
```

Comment:
Can't this be re-used between TokenizeOutputTestCase and ParseOutputTestCase? Is there any difference in the implementations?

Reply:
Nope, this was supposed to be moved to utils, removing the current test_case specific implementation.
Did just that over here: https://github.com/codecrafters-io/interpreter-tester/pull/14/commits/3b0d4a654d4a84cd907183a191a9e9b539f5bee2